### ISSUE TITLE
ariaOrientation for sliders is horizontal

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#116]
This is different from RIAW

**Comment by jessegreenberg** [comment]
ohms-law/SliderUnit is using VSlider now so hopefully the aria-orientation has the right default for that case. If it doesn't that should be fixed in AccessibleSlider. Assigning to myself to investigate.

**Comment by jessegreenberg** [comment]
I am seeing 
![image](https://user-images.githubusercontent.com/6396244/51947185-f53cbe00-23f1-11e9-9e1a-26fdbf9b5030.png)

now, this was likely fixed when we made HSlider and VSlider. The aria-orientation is set by Slider from the `orientation` option`.
```js
      this.initializeAccessibleSlider( valueProperty, this.enabledRangeProperty, this.enabledProperty,
        _.extend( {}, options, {
            ariaOrientation: options.orientation
          }
        ) );
```

Closing
